root/static/scripts/edit/URLCleanup.js

```diff
@@ -501,7 +548,11 @@ const CLEANUPS = {
       new RegExp("^(https?://)?(www\\.)?avexnet\\.jp/id/[a-z0-9]{5}/discography/product/[A-Z0-9]{4}-[0-9]{5}\\.html$", "i"),
       new RegExp("^(https?://)?(www\\.)?kingrecords\\.co\\.jp/cs/g/g[A-Z]{4}-[0-9]+/$", "i")
     ],
-    type: LINK_TYPES.discographyentry
+    type: LINK_TYPES.discographyentry,
+    validate: function (url, id) {
+      // avoid wikipedia being added as release-level discography entry
+      return !/^(https?:\/\/)?([^.\/]+\.)?wikipedia\.org\//.test(url);
```

Comment:
I wonder if it is good idea to add a `reject:` (or similar) directive, with a list of regexes that should be tested (and if found, rejected), but that wouldn’t cause cleanup or auto-selecting of the link type.